examples/gradle-kt-dsl/lib/src/test/kotlin/com/amazon/corretto/crypto/examples/AesGcm.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Kotlin source file was generated by the Gradle 'init' task.
+ */
+package com.amazon.corretto.crypto.examples
+
+import com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider
+import java.nio.charset.Charset
+import java.security.Security
+import javax.crypto.Cipher
+import javax.crypto.spec.GCMParameterSpec
+import javax.crypto.spec.SecretKeySpec
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+fun generateData(len: Int): ByteArray {
+    return ByteArray(len, Int::toByte)
+}
+
+class AesGcm {
+    @Test fun aesGcmEncryptDecrypt() {
+        Security.insertProviderAt(AmazonCorrettoCryptoProvider(), 1)
```

Comment:
nit: use `install()` method to align with installation/usage recommendations in our README

Reply:
Old habits. I'll fix this.